docs/IS_THIS_A_RAT.md

```diff
@@ -0,0 +1,25 @@
+There are websites that will claim SkyHanni is a rat.
+
+Those websites are wrong.
+
+Those websites might claim to know the mod has malicious code.
+
+Looking closer at the parts in the code they claim are malicious, we notice that their detection algorithms has a lot of flaws:
+
+for example:
+
+example 1
+
+Those websites might even claim their findings are powered by artifical intelligence. this is obviously wrong.
+
+Please don't trust websites that claim to know whether or not any mod can rat you or harm your computer.
+
+To download a mod safely, search for `github <mod name>` on the web.
```

Comment:
This line should be removed because it isn't necessarily true

Reply:
While this sentence is true for every known mod that exists for couple months, I agree, its still too broad.
what else would you say we should tell users how to find legit mods? just trust their big discord server, or search on Modrinth?